docs/resources/Guild.md

```diff
@@ -670,9 +670,6 @@ Create a new [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object for the gu
 
 Modify the positions of a set of [channel](#DOCS_RESOURCES_CHANNEL/channel-object) objects for the guild. Requires `MANAGE_CHANNELS` permission. Returns a 204 empty response on success. Fires multiple [Channel Update](#DOCS_TOPICS_GATEWAY/channel-update) Gateway events.
 
-> info
-> Only channels to be modified are required, with the minimum being a swap between at least two channels.
```

Comment:
Rather than removing the whole block, can we just remove the "minimum" part? It's probably still useful to know that you only have to send the channels you want to change in the array.